docs/UserGuide.md

```diff
@@ -14,165 +18,222 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 
-* Deletes the person at the specified `INDEX`.
-* The index refers to the index number shown in the displayed person list.
-* The index **must be a positive integer** 1, 2, 3, …​
+### Deleting a entry : `del`
+**NAME:**
+- `del` - delete an item or recipe
 
-Examples:
-* `list` followed by `delete 2` deletes the 2nd person in the address book.
-* `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
+**SYNOPSIS:**
+- `del` \<item name\> \[-r index\]
 
-### Clearing all entries : `clear`
+**DESCRIPTION:**
+- **deletes** the item in the inventory with the corresponding item name
+- **index:** 	if provided, deletes the <u>recipe</u> numbered <u>index</u>, 
 
-Clears all entries from the address book.
+**EXAMPLE:**
+- `del` <u>Bob’s 28th finger</u>
+- Deletes the **item** with the name of <u>Bob’s 28th finger</u>
 
-Format: `clear`
+### Finding an Entry: `find`
+**NAME:**
+- `find` - find an item
 
-### Exiting the program : `exit`
+**SYNOPSIS:**
+- `find` \<search string\>
 
-Exits the program.
+**DESCRIPTION:**
+- Displays items that match the search string, via the **description**.
 
-Format: `exit`
+**EXAMPLE:**
+- `find` -i <u>Bob’s 9000th crush</u>
+- Returns the item that match the item description: 
+<u>Bob’s 9000th crush</u>
 
-### Saving the data
+	(supports regex, eg. find -i <u>Bob’s ([1-9][0-9])(st\*nd\*rd\*th) crush</u>)
 
-AddressBook data are saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
 
-### Archiving data files `[coming in v2.0]`
+### View item `view`
+**NAME:**
+- `view` - view more details on an item
 
-_{explain the feature here}_
+**SYNOPSIS:**
+- `view` <item name> \[-r / -c / -d (default)/ -all\]
 
---------------------------------------------------------------------------------------------------------------------
+**DESCRIPTION:**
+- **-d:** 	returns item details
+- **-c:** 	returns all recipes that create the item
+- **-r:** 	returns all recipes that use the item
+- **-all:**	returns everything above
+
+**EXAMPLE:**
+- `view` <u>Bob’s bitten fingernail clipping</u> -r 
+- Returns all recipes that use <u>Bob’s bitten fingernail clipping</u>
+
+
+### Exiting the program : `bye`
+
+**NAME:**
+- `bye` - closes the application
+
+**SYNOPSIS:**
+- `bye`
+
+**DESCRIPTION:**
+- Closes the application. 
+
+### Saving the data
+
+Inventoryinator data is saved in the hard disk automatically after any command that changes the data.
+ There is no need to save manually. The format of save data is via `json file format`
 
 ## FAQ
 
 **Q**: How do I transfer my data to another Computer?<br>
-**A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous AddressBook home folder.
+**A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous Inventoryinator home folder.
```

Comment:
i think we should specify that they need to overwrite multiple files, since our storage for each class is in separate files